packages/plugin-compiler/src/main/kotlin/io/realm/kotlin/compiler/RealmModelLoweringExtension.kt

```diff
@@ -76,13 +75,13 @@ private class RealmModelLowering(private val pluginContext: IrPluginContext) : C
             // Unfortunately we cannot access the IR element of e uniformly across 1.9 and 2.0 so
             // leaving it as null. Hopefully the embedded cause will give the appropriate pointers
             // to fix this.
-            throw e.wrapWithCompilationException("Internal error in realm lowering", it, null)
+            throw CompilationException("Internal error in realm lowering $it", e, null)
         } catch (e: KotlinExceptionWithAttachments) {
             throw e
         } catch (e: Throwable) {
-            throw e.wrapWithCompilationException(
-                "Internal error in file lowering",
-                it,
+            throw CompilationException(
+                "Internal error in file lowering  $it",
```

Comment:
Unlikely, but the added space may break monitoring for someone

Reply:
Oh i didn't catch that! Fixing right when i have time later today :)